docs/reference/configuration/generic.md

```diff
@@ -97,6 +97,13 @@ Scenario:
 * Type: `required`
 * Default: `MESSAGE`
 
+## `SINK_CONNECTOR_SCHEMA_PROTO_UNKNOWN_FIELDS_VALIDATION_INSTRUMENTATION_ENABLE`
+
+Enable instrumentation for unknown fields validation. Instrumentation itself will introduces some overhead, so it should be enabled when needed.
```

Comment:
Enable instrumentation for unknown field validation for individual records. The instrumentation will introduce some overhead, So it should be enabled when needed.